app/views/dashboard/form/publish/_visibility_field.html.erb

```diff
@@ -1,6 +1,10 @@
-<div class="form-group">
-  <p class="label font-weight-bold mt-3"><%= WorkVersion.human_attribute_name(:visibility) %></p>
+<div class="keyline keyline--left">
+  <h2 id="<%= WorkVersion.human_attribute_name(:visibility).parameterize %>" class="h4">
+    <%= WorkVersion.human_attribute_name(:visibility) %>
+  </h2>
+</div>
 
```

Comment:
Maybe I missed this while I was out; are we not adding the ability to mint at DOI in this form for this release? That's fine if so. Just want to make sure it was intentionally omitted.

Reply:
right, we are omitting the minting DOI part, here is the [thread](https://github.com/psu-libraries/scholarsphere/issues/1261#issuecomment-1070984401) about that decision.